contrib/openshift/manifests/manifests.yaml

```
@@ -53,11 +53,10 @@ objects:
                 requests:
                   cpu: ${{INDEXER_CPU_REQS}}
                   memory: ${{INDEXER_MEM_REQS}}
-              lifecycle:
-                postStart:
-                  exec:
-                    command: ['sh', '-c', 'rm -rf /tmp/sha* /tmp/fetch.*']
-              command: ['clair']
+              command:
+                - sh
+                - '-c'
+                - rm -rf /tmp/sha* /tmp/fetch.* && clair
```

```suggestion
                - rm -rf /tmp/sha* /tmp/fetch.* && exec clair
```
Adding `exec` means `clair` will take over the shell process and not have it hanging around.